firebase/functions/package.json

```diff
@@ -19,7 +19,7 @@
     "eslint-utils": "^1.4.1"
   },
   "engines": {
-    "node": "8"
+    "node": "10"
```

Comment:
This is a pretty big change, with the potential to break a lot of code :)
Have you tested everything with node 10?

Reply:
mhh good questions, however I'm not really sure how to check properly. The only thing I can do is to check it somehow manually for the dev version of the app and look at all the places in firebase which should be updated.

But, we have to update to Node.js 10 by February 15, 2021 (#360 ). Since firebase is not allowing "old" functions which are written in Node.js 8 from then on.